assets/javascripts/commands_layout.js

```diff
@@ -1,13 +1,11 @@
-import $ from 'jquery'
 import AnchorJS from 'anchor-js';
 
 const anchors = new AnchorJS();
 
-$('.version-selects').change(function(e) {
-  document.location.href = $(e.target).find(":selected").val();
+const el = document.querySelector('.version-selects');
+el.addEventListener('input', function (e) {
+  document.location.href = e.target.value;
 });
 
-$(document).ready(function() {
-  anchors.add('#page-content-wrapper h1, #page-content-wrapper h2, #page-content-wrapper h3, ' +
-    '#page-content-wrapper h4, #page-content-wrapper h5');
-});
+anchors.add('#page-content-wrapper h1, #page-content-wrapper h2, #page-content-wrapper h3, ' +
+  '#page-content-wrapper h4, #page-content-wrapper h5');
```

Comment:
Minor: Perhaps we can use some formatting spaces to show that these strings are _very_ alike? E.g. one selector on each line, or something. Still as a selector, but formatted for easy scanning and understanding.

```suggestion
anchors.add(
  '#page-content-wrapper h1, ' +
  '#page-content-wrapper h2, ' + 
  '#page-content-wrapper h3, ' +
  '#page-content-wrapper h4, ' + 
  '#page-content-wrapper h5'
);
```

Reply:
This is unrelated to the PR, but I just thought of this change while reading. Not important that it goes in with this PR.

Reply:
Thanks again @olleolleolle 😍. Good point. I guess there is no `h5` (or `h1`) in `#page-content-wrapper`, so we should remove these.

```js
anchors.add('#page-content-wrapper h2, #page-content-wrapper h3, #page-content-wrapper h4');
```
or
```js
anchors.add(
  '#page-content-wrapper h2, ' + 
  '#page-content-wrapper h3, ' +
  '#page-content-wrapper h4'
);
```
would be the final result I am expecting. In either case, I would like to defer any change on this as possible as we can.

Reply:
Before this heuristic improvement, I want to apply the non-heuristic way in #737, @olleolleolle. It looks like more time need for me to change this snippet.